next/app/sitemap.ts

```diff
@@ -26,14 +26,10 @@ export default async function sitemap(): Promise<MetadataRoute.Sitemap> {
     let pageSize = 0;
 
     do {
-      const data = await drupal.doGraphQlRequest(
-        GET_SITEMAP_NODES,
-        {
-          page: page,
-          langcode: lang,
-        },
-        false, // We only want published nodes, so we disable authentication.
-      );
+      const data = await drupal.doGraphQlRequest(GET_SITEMAP_NODES, {
+        page: page,
+        langcode: lang,
+      });
```

Comment:
This is a good example, we used to disable authentication, now we can just leave the default which is to use the authentication. And because we are using the `drupal` client, the request will be done using the `next-api-viewer` role, which gets only published content